deploy.sh

```diff
@@ -5,8 +5,10 @@ set -e # Exit with nonzero exit code if anything fails
 REPO=`git config remote.origin.url`
 SSH_REPO=${REPO/https:\/\/github.com\//git@github.com:}
 SHA=`git rev-parse --verify HEAD`
+BRANCH=`git rev-parse --abbrev-ref HEAD`
 
-SOURCE_BRANCH="master"
+MASTER_BRANCH="master"
+DEVELOP_BRANCH="develop"
```

Comment:
These values are never used, but we should use them to check that we only compile and push if the script is called because of a modification of these two branches.

Reply:
[90a112a](https://github.com/tamarin-prover/manual/pull/101/commits/90a112aa1c78cbdcf68e096198ada7ec55393ee7) does that now.